tests/test.cpp

```diff
@@ -490,6 +490,15 @@ wxTestGLogHandler(const gchar* domain,
                   const gchar* message,
                   gpointer data)
 {
+    static int s_ignoreDebug = -1;
+    if ( s_ignoreDebug == -1 )
+    {
+        s_ignoreDebug = !wxGetEnv("G_MESSAGES_DEBUG", NULL);
+    }
```

Comment:
FWIW, you should just be able to do:

```
static int s_ignoreDebug = !wxGetEnv("G_MESSAGES_DEBUG", NULL);
```

Such variables get "initialized the first time control passes through their declaration":

https://en.cppreference.com/w/cpp/language/storage_duration#Static_local_variables

Reply:
Oops, yes, you're right, of course. I initially tested the value of the env var to check if `domain` argument was present in it (or if it was equal to "all"), but then decided it wasn't worth it and removed this code, but forgot to simplify it completely.